HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_03.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  def task3(log)
+    rasult_array = time_line(total_rows(log))
+    array_of_duration(rasult_array).size == 1 ? array_of_duration(rasult_array)[0] : array_of_duration(rasult_array)
+  end
+
+  private
+
+  def total_rows(log)
+    log.split("\n").select { |var| var.include?('Calling core with action') }
+  end
+
+  def time_line(array)
+    array.collect { |line| Time.parse(line) }
```

Comment:
why do you like to call everything "array..."))
```suggestion
    lines.collect { |line| Time.parse(line) }
```
```suggestion
    array.collect { |line| Time.parse(line) }
```